plugins/generator-1.14.4/forge-1.14.4/procedures/item_show_totemlike_animation.java.ftl

```diff
@@ -0,0 +1 @@
+Minecraft.getInstance().gameRenderer.displayItemActivation(${input$item});
```

Comment:
`Minecraft.getInstance()` will not work on dedicated servers and will crash them. I suggest adding world dependency and only calling this if world.getWorld().isRemote is true. Also, add client-side indicator icon similar to how existing client-side procedures do this.

Reply:
It has been fixed!